mypy.ini

```diff
@@ -1,2 +1,13 @@
 [mypy]
+show_error_codes = True
+exclude = (?x)(
+          setup\.py
+          ;          | tests/
+          | docs/
+          )
+
+[mypy-screenpy.*]
 disallow_untyped_defs = True
+
+[mypy-tests.*]
+ignore_missing_imports = True
```

Comment:
In most cases these settings do nothing for contributers since the pre-commit calls pylint and flake8 to inspect the code.   It's really only useful for anyone that want to only run mypy manually on their end for the entire project.  Handy, but not actually required by the codebase. I just found I was constantly adding them when I went to inspect the code manually locally.

Here is what the settings do:
- `show_error_code` :  includes the actual error code when running mypy rather than only the message.   I dunno why mypy doesn't do this by default.
- `exclude` : ignores the `setup.py` and the `docs` folder when running mypy manually against the entire project. 
- I initially ignored the tests folder then later commented it out. I can remove the comment. 
- `ignore_missing_imports` : ignores packages that don't have a typeshed to refer to.

Reply:
Forgot to mention moving `disallow_untyped_defs` down into `[mypy-screenpy.*]`.  This enforces the mypy rule specifically where it should be which is in the screenpy directory rather than the entire project.  This felt consistent with flake8 and pylint ignoring the tests also.